src/server/index.js

```diff
@@ -63,8 +63,10 @@ const port = process.env.DEV_APP_PORT || process.env.PORT
 
 // Don't rate limit heroku
 app.enable('trust proxy')
-if (!DEBUG && process.env.PUBLIC_DIR) {
-  app.use(express.static(process.env.PUBLIC_DIR, {
+
+// Serve static assets
+if (existsSync(process.env.ASSETS_DIR)) {
```

Comment:
is this an intentional change from PUBLIC_DIR => ASSETS_DIR?
Can you explain why it's ASSETS_DIR and not PUBLIC_DIR?

Also, why are we removing the DEBUG condition?  I suppose some containerization deployments have simplicity benefits in serving static assets through node rather than a static file server?  If so, I think we should at least have an env var that can disable this (no matter the presence of ASSETS_DIR).

Reply:
Also part of the heroku PR. 😄Can we move this discussion over to #1053?